explainer.md

```diff
@@ -278,52 +159,66 @@ class CustomCheckbox extends HTMLElement {
   attributeChangedCallback(name, oldValue, newValue) {
     switch(name) {
     case "checked":
-      this.accessibleNode.checked = (newValue !== null);
+      this.shadowRoot.ariaChecked = (newValue !== null);
     }
   }
 
   connectedCallback() {
-    // When the custom checkbox is inserted in the page,
-    // ensure the checked state is in sync with the checked attribute.
-    this.accessibleNode.checked = this.hasAttribute("checked");
+    // Set the default semantics for the custom element
+    // when it is inserted in the page.
+    this.shadowRoot.role = "checkbox";
+    this.shadowRoot.checked = this.hasAttribute("checked");
```

Comment:
I don't think this is how you would do this.

Reply:
Typo: should have been `.ariaChecked`.